test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -458,6 +458,7 @@ def setUpClass(cls):
             else:
                 cls.logger.error("Failed to save custom setup")
                 raise Exception("Failed to save custom setup")
+            cls().add_mgrs_opers(cls.server)
```

Comment:
this looks kind of hack-y .. you are creating an instance of the class via cls() to call a non-classmethod from a classmethod. why not just make add_mgrs_opers() a classmethod? Then you can just call it as "cls.add_mgrs_opers()" here. Inside setUp() for the normal code path, you can just call it as self.add_mgrs_opers(). You'll have to change "server" to "cls.server" inside the method

Reply:
I tried this yesterday and I was getting some error so I did not debug further. Maybe I was surely doing something wrong. I have made the changes as per your suggestion. Please review the latest commit.